nh_eobs_mental_health/security/ir.model.access.csv

```diff
@@ -7,4 +8,14 @@ dev_access_nh_clinical_notification_clinical_review,developer:access_nh_clinical
 admin_access_nh_clinical_notification_clinical_review,admin:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_admin,1,1,1,0,0
 adt_access_nh_clinical_notification_clinical_review,adt:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_adt,1,1,1,0,0
 hca_access_nh_clinical_notification_clinical_review,hca:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_hca,1,1,1,0,0
-doctor_access_nh_clinical_notification_clinical_review,doctor:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_doctor,1,1,1,1,1
\ No newline at end of file
+doctor_access_nh_clinical_notification_clinical_review,doctor:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_doctor,1,1,1,1,1
+
+base_access_nh_clinical_notification_clinical_review_frequency,restrict:access_nh_clinical_notification_clinical_review_frequency,model_nh_clinical_notification_clinical_review_frequency,nh_clinical.group_nhc_base,0,0,0,0,0
+nurse_access_nh_clinical_notification_clinical_review_frequency,nurse:access_nh_clinical_notification_clinical_review_frequency,model_nh_clinical_notification_clinical_review_frequency,nh_clinical.group_nhc_nurse,1,1,0,0,1
```

Comment:
Checked this and (after modifying the report to show actions triggered from partials) the *clinical review* and *clinical review frequency* tasks both appeared in the *Actions Triggered* section.